homeassistant/components/opentherm_gw.py

```diff
@@ -152,11 +189,17 @@
             gw_vars.OTGW_VREF,
         ]
     }
+    binary_sensors = []
     sensors = []
     for var in monitored_vars:
         if var in sensor_type_map[COMP_SENSOR]:
             sensors.append(var)
+        elif var in sensor_type_map[COMP_BINARY_SENSOR]:
```

Comment:
Are the vars in the maps exclusive?

Reply:
Yes, everything that can either be 0 or 1 is a binary sensor, everything else is a normal sensor.